fastlane/lib/fastlane/actions/pilot.rb

```diff
@@ -9,6 +9,7 @@ def self.run(values)
         values[:changelog] ||= changelog if changelog
 
         values[:ipa] ||= Actions.lane_context[SharedValues::IPA_OUTPUT_PATH]
+        values[:ipa] = File.expand_path(values[:ipa]) if values[:ipa]
```

Comment:
@taquitos the only issue with this approach is that if values[:ipa] is passed in, the path won't be expanded.

Maybe something like this?
```
ipa_path_from_context = Actions.lane_context[SharedValues::IPA_OUTPUT_PATH]
values[:ipa] ||= ipa_path_from_context
values[:ipa] = File.expand_path(values[:ipa]) if values[:ipa]
```